src/dswx_sar/fuzzy_value_computation.py

```diff
@@ -114,7 +115,7 @@ def smf(values, minv, maxv):
         diff = maxv - minv
         number_strange = np.nansum(diff <= 0)
         if number_strange > 0:
-            logger.info(f'{number_strange} pixels have minimum values' \
+            logger.info(f'{number_strange} pixels have minimum values'
```

Comment:
```suggestion
            logger.info(f'{number_strange} pixels have minimum values '
```
To add space between `values` and `larger` in the line below